src/components/sidenav/demoBasicUsage/index.html

```diff
@@ -4,11 +4,10 @@
   <section layout="row" flex>
 
     <md-sidenav
-        class="md-sidenav-left"
+        class="md-sidenav-left md-whiteframe-4dp"
```

Comment:
That seems to be covered in the md-whiteframe docs. Do we need to cover both usages of md-whiteframe in the md-sidenav docs? It seems like it only adds to the confusion and complexity of learning about sidenavs.